dev/config/pyramid_oereb.yml.mako

```diff
@@ -89,6 +89,9 @@ pyramid_oereb:
     # not set it will assume that only one TOC page exists, and this can lead to wrong numbering in the TOC, which
     # will be fixed by a second PDF extract call that has an impact on performance.
     compute_toc_pages: false
+    # The calculation of the TOC pages is very detailed in pyramid_oereb. However there are "earlier" page breaks, that are
```

Comment:
I would change the text slightly: "The calculation of the number of TOC pages is very detailed in pyramid_oereb. However, it is not complete, and page breaks occur slightly earlier in page than expected. With setting the value "page_break_difference" (default: 10px), the apparent (not the real) page height is reduced in order to account for this issue."

Reply:
Done in this commit: https://github.com/openoereb/pyramid_oereb/pull/2047/commits/7a9de6900e339374716e87bce18ad10548782823